src/components/CompareResults/PaginatedCompareResults.tsx

```diff
@@ -0,0 +1,180 @@
+import { useState } from 'react';
```

Comment:
Most of the code in this component came directly from an example in the material ui docs: https://mui.com/material-ui/react-table/#custom-pagination-actions